packages/overlay/stories/overlay.stories.ts

```diff
@@ -631,13 +631,13 @@ export const deepNesting = (): TemplateResult => {
         ${storyStyles}
         <sp-theme
             color=${outter}
-            theme=${window.__swc_hack_knobs__.defaultThemeVariant}
+            theme=${window.__swc_hack_knobs__.defaultSystemVariant}
```

Comment:
We should probably document on `system` for clarity to consumers here.

Reply:
Ok! Let's add `system=${window.__swc_hack_knobs__.defaultSystemVariant}` with `theme=${window.__swc_hack_knobs__.defaultSystemVariant}`